compiler/codegen/OMRTreeEvaluator.cpp

```diff
@@ -147,6 +147,24 @@ bool OMR::TreeEvaluator::instanceOfOrCheckCastNeedSuperTest(TR::Node * node, TR:
    return false;
    }
 
+TR_GlobalRegisterNumber 
+OMR::TreeEvaluator::getHighGlobalRegisterNumberIfAny(TR::Node *node, TR::CodeGenerator *cg)
+   {
+    //No need for register pairs in 64-bit mode
+    if (TR::Compiler->target.is64Bit())
+        return -1;
+
+    //if the node itself doesn't have a type (e.g passthrough) we assume it has a child with a type
+    //However we keep track of the initial node as it will contain the register information.
+    TR::Node *rootNode = node;
+
+    while (node->getType() == TR::NoType)
+        node = node->getFirstChild();
```

Comment:
Are you certain this will always find a node with a type by traversing the first child?  I mean, it seems reasonable to assume and I can't think of a counter example, but the loop proceeds always assuming its going to find a valid first child (i.e., no check for `node` null-ness).